owner-onboarding-server/src/main.rs

```diff
@@ -95,6 +100,10 @@ struct Settings {
     owner_addresses_path: AbsolutePathBuf,
 
     report_to_rendezvous_endpoint_enabled: bool,
+
+    // in seconds
+    registration_period: Option<String>,
```

Comment:
Is there a specific reason you don't do `Option<u32>`?

Reply:
Yes, to the best of my knowledge serde doesn't have a way to check that u32 must be positive while loading the settings from the file, even if we are requesting a u32, thereby my tests show, that if I put a minus something it will be wrapped to a random positive integer (on target debug and release).

I haven't found a way to catch when serde does this. There are some crates to allow these types of checks but they didn't look stable enough and we have a way to check this without external crates.

Reply:
Ah, I would've expected signed integers to cause an error. That's unfortunate, but fair enough then.